src/ontology/cl-edit.owl

```diff
@@ -4206,6 +4207,7 @@ AnnotationAssertion(owl:deprecated obo:CL_0000074 "true"^^xsd:boolean)
 
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GO:0002065") Annotation(oboInOwl:hasDbXref "https://orcid.org/0000-0001-5208-3432") obo:IAO_0000115 obo:CL_0000075 "A columnar/cuboidal epithelial cell is a cell usually found in a two dimensional sheet with a free surface. Columnar/cuboidal epithelial cells take on the shape of a column or cube.")
 AnnotationAssertion(rdfs:label obo:CL_0000075 "columnar/cuboidal epithelial cell")
+EquivalentClasses(obo:CL_0000075 ObjectIntersectionOf(obo:CL_0000066 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000483)))
```

Comment:
This should be removed, it is saying that all epithelial cells that are part of an epithelium are columnar/cuboidal epithelial cells.

Reply:
These commits are not part of this PR.

Reply:
Are you having axioms added sporadically? Have you noticed if it is a protege problem or Github Desktop problem?

Reply:
I am not sure. I will check on protoge